packages/feedback/typings/FeedbackForm.d.ts

```diff
@@ -16,6 +16,7 @@ export interface FeedbackFormProps {
   onFeedbackSent?: (feedback?: { [key: string]: any }) => void;
   faceOptions?: FaceOption[];
   aboutOptions?: AboutOption[];
+  customPlaceholder?: string;
```

Comment:
we need a corresponding change to the documentation as well

Reply:
Ah yes, thank you. I meant to do that.

I noticed the URL in the README for Feedback results in "Page not Found" - (https://availity.github.io/availity-react/components/feedback). 

Am I okay to change that to the specific URL: https://availity.github.io/availity-react/components/feedback/index